packages/libs/react-components/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,30 @@
+import { Heading } from '../Typography';
+
+import { StyledCard } from './styles';
+
+import { VariantProps } from '@stitches/react';
+import React, { FC } from 'react';
+
+export interface ICardProps {
+  children: React.ReactNode;
+  fullWidth?: VariantProps<typeof StyledCard>['fullWidth'];
+  stack?: boolean;
```

Comment:
nit: This should never be anything outside of true or false, but since this is tied to a variant, it's generally best to make the type also tied to the variant as well. So I would type it like fullWidth